homeassistant/components/hassio/__init__.py

```diff
@@ -124,6 +125,21 @@
 }
 
 
+@bind_hass
+async def async_get_addon_info(hass: HomeAssistantType, addon_id: str) -> dict:
+    """Return add-on info.
+
+    The addon_id is a snakecased concatenation of the 'repository' value
```

Comment:
I don't think that we should document this here? We're going to add more methods here eventually and you want to add this to every doc string ? 

We should also call this slug. Everywhere _but_ `config.json` we use `slug` to refer to `<repo>-<addon slug>`, so we should do that here too. We should expose exactly how the HTTP API is, including the used names. 

If we want to change names, we start with the HTTP API which is the source of truth, then the documentation and then methods that expose that API.

Reply:
We don't use `slug` everywhere today. We also use eg `addon`.

I think the naming isn't consistent today and very confusing. But I'd like to get this PR merged soon.